### DISCUSSION TITLE
Mardown Table w/ Component Rows

### DISCUSSION THREAD

**Discussion by allcentury** [mdx-js/mdx#1456] [Q&A] [answered]
I have a table where I want some of the rows to have a border (kinda like this)...

<img width="169" alt="Screen Shot 2021-02-02 at 1 45 42 PM" src="https://user-images.githubusercontent.com/5561166/106667337-a6383f00-655d-11eb-95b6-983e657bc804.png">

So what I think I want is to be able to tag certain `<tr>`'s with a class or attribute so I can use a CSS selector.

However this doesn't work:

```jsx
export const BlueRow = () => <div class="blue-row">G | H</div>

Col | Col2
--|---
A | B
C | D
E | F
<BlueRow />
```

Is there anyway to do what I'm suggesting?

**Comment by ChristianMurphy** [comment, marked as answer]
Github flavored markdown doesn't have a mechanism for customizing rows.
However, MDX just support JSX, and styles can be added using HTML tags and/or React components to generate the table.
